mantidproject/mantid
# [PD461]  Update reference file due to change in fitting

**Description of work.**

- Original Gitlab Task: [PD461](https://code.ornl.gov/sns-hfir-scse/diffraction/powder/powder-diffraction/-/issues/461)



Due to recent change in the fitting, the reference file need to be updated.
The actual change is not visible, but it does got picked-up by `CompareWorkspace`.

**To test:**

Build `mantid` on a linux platform, then go to the build folder and issue `./systemtest -R "PG3StripPeaks"`.
The corresponding system test should pass with the new reference file.










---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
